content/docs/other-guides/freq-ask-questions.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Frequently Asked Questions"
+description = ""
+weight = 110
++++
+
+
+### Is deploying Kubeflow with Helm supported?
+
+No, Kubeflow only provides kustomize packages for Kubeflow applications. So deploying Kubeflow isn't supported except via kustomize.
+
+Kubeflow makes use of [kustomize](https://kustomize.io/) to help customize YAML
```

Comment:
Suggestion: "to help customize Kubeflow for different platforms and configurations. 

I think you can then drop the sentence "With kustomize..."

Reply:
thanks jeremy: I made changes.